components/footer/index.js

```diff
@@ -1,20 +1,133 @@
 import styled from "styled-components"
+import { AiOutlineTwitter, AiFillMediumSquare, AiFillGithub } from 'react-icons/ai'
+import { FaDiscord } from 'react-icons/fa'
+import { SiNotion } from 'react-icons/si'
+import Image from 'next/image'
 
 const FooterSection = styled.div`
-background: #202020;
-color: #fff;
-height: 60px;
+background: var(--footer-background);
+padding: 50px 0 75px 0;
+`
+
+const FooterContainer = styled.div`
```

Comment:
we should probably start moving these styles into their own file either as css or in a styles js file, something like `FooterStyles.js` to make it a bit more manageable

Reply:
Agreed.